test/load/cmd/client.go

```diff
@@ -0,0 +1,72 @@
+package cmd
+
+import (
+	"crypto/tls"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+
+	"github.com/mitchellh/go-homedir"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/client-go/rest"
+	restclient "k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func newClient() (*restclient.RESTClient, *restclient.Config, error) {
+	kubeconfig, err := contextConfig()
```

Comment:
Should we rename this `kubeconfig` function variable to something else since there's a global variable also named `kubeconfig`? Maybe `clientconfig` or something?